test/powershell/Language/Scripting/TryCatch.Tests.ps1

```diff
@@ -587,7 +587,7 @@ Describe "Test try/catch" -Tags "CI" {
               }
             }
 
-            ($a -eq "test passed") | Should -Be $true
+            ($a -eq "test passed") | Should -BeTrue
```

Comment:
should be $a | should -be "test passed"
note: ok to limit this to pure syntax conversions